ee/app/livechat-enterprise/server/hooks/onSaveVisitorInfo.js

```diff
@@ -0,0 +1,9 @@
+import { callbacks } from '../../../../../app/callbacks';
+import { LivechatEnterprise } from '../lib/LivechatEnterprise';
+
+callbacks.add('livechat.saveInfo', async (room, user) => {
+	if (!room || !user) {
+		return;
+	}
+	LivechatEnterprise.savePriorityDataOnRooms(room, user);
```

Comment:
Let's check the new and the old priority value here, then we don't need to do anything if the values are the same.